web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.styles.tsx

```diff
@@ -0,0 +1,99 @@
+import { makeStyles } from '@mui/styles';
+
+export const useBuyCreditsModalStyles = makeStyles(theme => ({
```

Comment:
These are not new styles, just old ones extracted for better readability in `BuyCreditsModal`.